app/src/main/res/values-de/strings.xml

```diff
@@ -361,4 +361,5 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="expand_collapse">Erweitern / Einklappen</string>
     <string name="essential">Essenziell</string>
     <string name="append_human_readable_uuid_to_new_tasks">Menschlich lesbare UUID neuen Aufgaben hinzufügen</string>
+    <string name="password">Passwort</string>
```

Comment:
Please don't add translations. It will be overwritten by Crowdin anyway. Instead add translations on Crowdin once merged

Reply:
Well I thought this way we have a solution till a Crowdin-Change is made. No idea how Crowdin works.